Tests/SonarScanner.MSBuild.Tasks.UnitTests/GetAnalyzerSettingsTests.cs

```diff
@@ -405,6 +428,31 @@ private static GetAnalyzerSettings CreateConfiguredTestSubject(AnalysisConfig co
             return testSubject;
         }
 
+        private static AnalysisConfig CreateMergingAnalysisConfig(string language, string qpRulesetFilePath) =>
```

Comment:
Miscellaneous: helper method to avoid repeating the same code for a config that is configured to make merging occur.